prov/gni/src/gnix_tags.c

```diff
@@ -930,6 +930,7 @@ void _gnix_remove_tag(
 		struct gnix_tag_storage *ts,
 		struct gnix_fab_req *req)
 {
+	GNIX_DEBUG(FI_LOG_EP_DATA, "\n");
```

Comment:
This debug statement seems like it could get a little noisy.  Personally, I think it should be removed.